README.md

```diff
@@ -114,22 +128,42 @@ func main() {
 }
 ```
 
-### Run FileUploader
+**1. Create a test file containing data:**
+
+On Linux systems, you may do this with `dd`:
+```
+dd if=/dev/urandom of=/tmp/testdata bs=2048 count=10
+```
+
+**2. Run FileUploader with the following commands:**
```

Comment:
Feedback on these commands very welcome, this is what worked for me. I'm not sure the usual Go way of referring to these operations.

For now, this is not intended to be a comprehensive discussion. Just the basics for someone generally familiar with the language and tools.